src/components/vuestic-components/vuestic-breadcrumbs/VuesticBreadcrumbs.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <vuestic-widget class="no-padding no-v-padding">
```

Comment:
On some pages breadcrumbs and main content have different width
pages:
- Progress Bars
- Buttons
- Icons
- Spinners
- grid
- modals
- maps

Reply:
What is the proper usage of vuestic-widget. Inside <div class="row"> and with class "col-sm-12" or without anything? Components use these approaches 50/50

Reply:
@smaliankou the proper approach is

.row
   .col-xx
      <vuestic-widget>

Just like Form Elements page